apps/sms/test/marionette/mocks/mock_message_thread.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
Yeah, I wish we didn't have to inject the dependencies to Marionette so this class could only focus on the thread generation.
I'm not a huge fan of duplicating the logic in multiple the tests, though. Especially when the generation takes 30 LOC. Reducing the message generation from 30 LOC to 2, looks like an improvement.
Moreover, I think `injectDependencies` and `generateThread` give more explanation about the intention behind these LOC. 

What do you think?